### PR TITLE
Mobile portfolio

### PR DESCRIPTION
**Pull request by RHK-MICROVERSE**
### In this pull request

- [ ] We implemented mobile sections of my portfolio
- [ ] Added readme
- [ ] No linters errors

### CODE REVIEW

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @RHK-MICROVERSE, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️

**I CAN SEE THAT MOST OF THE CHANGES WERE NOT IMPLEMENTED IN THIS PULL REQUEST WHICH IS MAKING IT DIFFICULT TO GIVE QUALITY REVIEWS. KINDLY ENSURE THAT ALL YOUR CHANGES ARE REFLECTED IN THE PULL REQUEST YOU ARE SUBMITTING.**

- [x] KIndly ensure you wrap your entire toolbar (logo and hamburger) in a header tag that is semantic enabling screen readers and search engines to understand what each content does 👍🏽. Get more info [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header)

- [x] Kindly ensure that only your hamburger is wrapped in a nav tag as it will contain your navigation elements eventually. Your logo should be part of the header tag and not the nav tag

- [x] Kindly ensure that the content of your toolbar is properly aligned together. Right now they are not properly aligned. Add `align-items: center` to properly align the contents to the center 

your design 👇🏽 
image

Figma design 👇🏽 
image

- [x] Kindly ensure you use the right background color in your headline section. Currently, your background color is black but the figma design is #1C1A19. KIndly check it out.

your design 👇🏽 
image

figma design 👇🏽 
image

_Check the comments under the review._


- [x] Kindly ensure that your list of icons are wrapped in a `<ul>` tag and `<li>`  tag per icon; and each icon has a link (a tag).

e.g 

 ```
   <ul>
         <li> 
              <a> icon </a> 
         </li>
     </ul>
```

- [x] Kindly ensure that your main title (e.g., "I'm Marianne ...") is an `h1` tag. If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info,

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -39,10 +41,12 @@ <h3 class="activity">I am a Full Stack Developer</h2>
...
     <div class="social-media">
-      <li> <img src="./images/git-icon.png" class="icon git"></a></li>
-      <li> <img src="./images/linkedin-icon.png" class="icon linkedin"></a></li>
-      <li> <img src="./images/victory-icon.png" class="icon victory"></a></li>
-      <li> <img src="./images/m-icon.png" class="icon gmail"></a></li>
+      
+      <div> <img src="./images/git-icon.png" class="icon git"></a></div>
+      <div> <img src="./images/linkedin-icon.png" class="icon linkedin"></a></div>
+      <div> <img src="./images/victory-icon.png" class="icon victory"></a></div>
+      <div> <img src="./images/m-icon.png" class="icon gmail"></a></div>
```
Comment (mahbubul14) [inline index.html]: - [x] As the previous reviewer suggested, Kindly ensure that your list of icons is wrapped in a `<ul>` tag and `<li>` tag per icon; and that each icon has a link (a tag).

e.g:

```
<ul>
        <li>
            <a> icon </a>
        </li>
</ul>
```

File: index.html [hunk 2]
```diff
@@ -17,18 +17,20 @@
...
+    </nav>
+  </header>
 
   <section class="container">
     <div class="primary-text">
-      <h2 class="greetings">Hey there. I'm Rassel</h2>
+      <h1 class="greetings">Hey there. I'm Rassel</h1>
     </div>
     <div class="support-text">
       <h3 class="activity">I am a Full Stack Developer</h2>
```
Comment (mahbubul14) [inline index.html:36]: - [x] Kindly give a proper space between the two sections and match it with the Figma design. Here is your design looks now:

readme

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @RHK-MICROVERSE,

Great work on making the changes requested by the previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

# Required Changes ♻️

- [x] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 3]
```diff
@@ -17,18 +17,20 @@
...
+      <div class="hum">
+        <img src="./images/hum.png" alt="">
+      </div>
+    </nav>
+  </header>
 
   <section class="container">
     <div class="primary-text">
-      <h2 class="greetings">Hey there. I'm Rassel</h2>
+      <h1 class="greetings">Hey there. I'm Rassel</h1>
```
Comment (AdnanOlarmmi) [inline index.html:33]: - [ ] [OPTIONAL] Kindly reduce the space a bit to make it more pixel-perfect, as you shall be doing in your soon coming life-changing future job 😀.

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @RHK-MICROVERSE,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Nice design.
- Professional README.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**